worlds/sm64ex/docs/setup_en.md

```diff
@@ -59,6 +59,56 @@ If everything worked out, you will see a textbox informing you the connection ha
 To play offline, first generate a seed on the game's settings page.
 Create a room and download the `.apsm64ex` file, and start the game with the `--sm64ap_file "path/to/FileName"` launch argument.
 
+# Using Batch Files to play offline and MultiWorld games
+
+As an alternative to launching the game with sm64pclauncher, it is also possible to launch the completed build with the use of Windows batch files. This has the added benefit of streamlining the join process so that manual editing of connection info is not needed for each new game. However, you'll need to be somewhat comfortable with creating and using batch files.
+
+IMPORTANT NOTE: The remainder of this section uses copy-and-paste code that assumes you're using the US version. If you instead use the Japanese version, you'll need to edit the EXE name accordingly by changing "sm64.us.f3dex2e.exe" to "sm64.jn.f3dex2e.exe".
+
+### Making an offline.bat for launching offline patch files
+
+Open Notepad. Paste in the following text: `start sm64.us.f3dex2e.exe --sm64ap_file %1 --skip-intro`
+
+Go to File > Save As...
+
+Navigate to the folder you selected for your SM64 build when you followed the Build guide for SM64PCLauncher earlier. Once there, navigate further into `build` and then `us_pc`. This folder should be the same folder that `sm64.us.f3dex2e.exe` resides in. 
+
+For "File name:" name it as: `"offline.bat"` . THE QUOTE MARKS ARE IMPORTANT! Otherwise it will create a text file instead ("offline.bat.txt") which won't work as a batch file until you alter the extension properly.
+
+Now you should have a "offline.bat" file with a gear icon in the same folder as your "sm64.us.f3dex2e.exe". (If the icon is instead a notepad, you saved it as a TXT file instead and need to change the extension to BAT). Right click that offline.bat file and choose Send To > Desktop (Create Shortcut).
+
+The batch file you just created and made a shortcut for is set up to accept patch files dragged into it. Start an offline singleplayer game if you haven't already, download the `.apsm64ex` patch file that was prepared by the seed generation, and drag-n-drop that onto the batch file to open the game and start playing.
+
+NOTE: When playing offline patch files, a `.save` file is created in the same directory as your patch file, which contains your save data for that seed. Don't delete it until you're done with that seed.
+
+### Making an online.bat for launching online Multiworld games
+
+The steps here are very similar, as you will be making a batch file in the same location as the offline one. However, the text you put into the batch file is different, and you won't be dragging patch files onto it.
+
+Use the same steps as before to open Notepad and paste in the following:
+
+`set /p port="Enter port number of room - "`
+
+`set /p slot="Enter slot name - "`
+
+`start sm64.us.f3dex2e.exe --sm64ap_name "%slot%" --sm64ap_ip archipelago.gg:%port% --skip-intro`
```

Comment:
```suggestion
`start sm64.us.f3dex2e.exe --sm64ap_name "%slot%" --sm64ap_ip archipelago.gg:%port%`
```
You hardcode the Server IP here. Might be nicer if it is provided from the user as well.